Felienne/hedy
# Capitalizes first letter of username

**Description**
In this PR we make a small change where we capitalize the first letter of the username is most places a username occurs. This makes it look a bit nicer as all names should start with a capitalized letter.

**Fixes**
This PR fixes #3224.

**How to test**
Nothing to test, just verify that the first letter of your username is capitalized when logged in.
